app/features/onboarding/onboarding_controller.rb

```diff
@@ -127,6 +127,18 @@ class OnboardingController < ControllerBase
     # 3) If the data is not valid, display an error message
     post "#{HOME}/initial_clone_user" do
       if valid_params?(params, post_parameter_list_for_clone_user_validation)
+        api_token = params[:clone_user_api_token]
+        api_token.strip!
```

Comment:
Let's append the `.strip` directly to line `:130` without the `!` to make it easier to understand for new contributors 👍